test/unit/ui/marker.test.js

```diff
@@ -16,31 +16,60 @@ function createMap() {
 }
 
 test('Marker', (t) => {
+
     t.test('constructor', (t) => {
-        const el = window.document.createElement('div');
-        const marker = new Marker(el);
-        t.ok(marker.getElement(), 'marker element is created');
+        const marker = new Marker();
+        t.ok(marker, 'marker is created');
```

Comment:
the original test case here isn't affected by the introduction of the anchor option so best to leave it as is and add new tests if you have new functionality to test.